acceptance-tests/src/test/java/uk/gov/di/test/pages/BasePage.java

```diff
@@ -164,6 +164,19 @@ public String getPageHeading() {
         return findElement(By.cssSelector("h1")).getText().trim();
     }
 
+    public String getLockoutText() {
+        return findElement(By.xpath("//*[normalize-space()='wait 2 hours, then try again']")).getText().trim();
+    }
+
+    public String getLockoutTexts() {
+        return findElement(By.xpath("//*[normalize-space()='Wait 2 hours, then try again.']")).getText().trim();
+    }
+
+    public String getRetryText() {
+//        String actualText = findElement(By.xpath("//*[@class='govuk-body strategic-app-retry-options']")).getAttribute("You can also wait 2 hours, then try again.");
```

Comment:
Hi @Vivektiwari11, just wondering if we should remove this commented code from the test?

Reply:
Updated please review @gtvj